### PR TITLE
Added ESLint, and some relevant rules.

### PR DESCRIPTION
**Pull request by SammyIsra**
Additionally, corrected Linter rules, and moved around some imports.

Sorry about the code review you're about to do! Let me know if you want me to clear anything up.

### CODE REVIEW

**Comment by SammyIsra** [comment]
Last commit broke it, as @allygator pointed out, because of my mistake. This should fix it.

File: src/components/Calc.js [hunk 1]
```diff
@@ -1,167 +1,254 @@
...
+  }
+
+  handleChange = name => event => {
+    this.setState({ [name]: event.target.checked });
+  };
+
+  enterPressed(event) {
+    var code = event.keyCode || event.which;
+    if (code === 13) {
+      //13 is the enter keycode
```
Comment (allygator) [inline src/components/Calc.js]: this comment should be on the same line as the code

File: src/components/Calc.js [hunk 2]
```diff
@@ -90,17 +99,17 @@ class Calc extends Component {
         this.setState({completedCount:number});
     }
 
-    showReputations(e) {
+    showReputations() {
         //console.log("Submitted");
-        this.setState({isSubmitted:true, showSearch:false})
-        this.setState({submittedName:this.state.name, submittedRealm:this.state.realm, submittedRegion:this.state.region, isCompleted:this.state.completed, });
+        this.setState({isSubmitted:true, showSearch:false});
+        //this.setState({submittedName:this.state.name, submittedRealm:this.state.realm, submittedRegion:this.state.region, isCompleted:this.state.completed, });
```
[the remark this answers is not in the document]
Comment (SammyIsra) [reply src/components/Calc.js]: Addressed: https://github.com/allygator/wowReputationCalculator/pull/29/files#diff-58dbce7f1e2afb8215e1c426f7a4c76bR104

**Comment by SammyIsra** [comment]
Changes have been addressed 😄

**Comment by SammyIsra** [comment]
Oh wait, no it doesn't work

**Pull request closed without merging**